### ISSUE TITLE
Can I use codon with pytorch and other deep learning libraries for faster multithreading ?

### ISSUE THREAD

**Issue by UygarUsta99** [exaloop/codon#285]
As I said in the title,is it possible to use deep learning frameworks like pytorch to take advantage of codon's multithreading ? Also would there be any problems using the gpu with pytorch ?

**Comment by inumanag** [comment]
Sadly, not yet—Pytorch is not natively supported by Codon (only through the Python bridge). You can use Pytorch via `from python import torch`, but you won't be able to get any speedups that way. You might be able, though, to improve and/or speed-up callbacks written in Python with Codon.

**Comment by inumanag** [comment]
Duplicate of #349.
